examples/email-recovery/src/pages/index.tsx

```diff
@@ -0,0 +1,239 @@
+
+    // An example of possible options can be found here:
+    // https://www.w3.org/TR/webauthn-2/#sctn-sample-registration
+    const attestation = await getWebAuthnAttestation({
+      publicKey: {
+        authenticatorSelection: {
+          residentKey: "preferred",
+          requireResidentKey: false,
+          userVerification: "preferred",
+        },
+        rp: {
+          id: "localhost",
+          name: "Turnkey Federated Passkey Demo",
+        },
+        challenge,
+        pubKeyCredParams: [
+          { type: publicKey, alg: es256 },
+          { type: publicKey, alg: rs256 },
+        ],
+        user: {
+          id: authenticatorUserId,
+          name: data.authenticatorName,
+          displayName: data.authenticatorName,
+        },
+      },
+    });
+
+    const client = new TurnkeyClient(
+      {
+        baseUrl: process.env.NEXT_PUBLIC_BASE_URL!,
+      },
+      iframeStamper
+    );
+
+    // TODO: switch this to be `client.recoverUser`!
+    // Similar in spirit though, we're adding a new authenticator.
+    const signedRequest = await client.stampCreateAuthenticators({
+      type: "ACTIVITY_TYPE_CREATE_AUTHENTICATORS_V2",
+      timestampMs: String(Date.now()),
+      organizationId: initRecoveryResponse.organizationId,
+      parameters: {
+        userId: initRecoveryResponse.userId,
+        authenticators: [
+          {
+            authenticatorName: data.authenticatorName,
+            challenge: base64UrlEncode(challenge),
+            attestation: attestation,
+          },
+        ],
+      },
+    });
+
+    // TODO: instead of simply alerting, POST the request to Turnkey to complete recovery.
+    console.log("SUCCESS", signedRequest);
+    alert("SUCCESS! Request SIGNED!");
+  };
+
+  return (
+    <main className={styles.main}>
+      <a
+        href="https://www.turnkey.com"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <Image
+          src="/logo.svg"
+          alt="Turnkey Logo"
+          className={styles.turnkeyLogo}
+          width={100}
+          height={24}
+          priority
+        />
+      </a>
+
+      <Recovery
+        setIframeStamper={setIframeStamper}
+        iframeUrl={process.env.NEXT_PUBLIC_RECOVERY_IFRAME_URL!}
+        turnkeyBaseUrl={process.env.NEXT_PUBLIC_BASE_URL!}
+      ></Recovery>
+
+      {!iframeStamper && <p>Loading...</p>}
+
+      {iframeStamper &&
+        iframeStamper.publicKey() &&
+        initRecoveryResponse === null && (
+          <form
+            className={styles.form}
+            onSubmit={initRecoveryFormSubmit(initRecovery)}
+          >
+            <label className={styles.label}>
+              Email
+              <input
+                className={styles.input}
+                {...initRecoveryFormRegister("email")}
+                placeholder="Email"
+              />
+            </label>
+            <label className={styles.label}>
+              Target Public Key (FYI)
```

Comment:
naive q: what does `FYI` refer to here?

Reply:
Oh that's just "For You Information". Because it's not really needed in the UI. But I wanted to show it in the demo because it's interesting to see I think?